packages/grant-explorer/src/store.ts

```diff
@@ -71,6 +79,10 @@ export const useCartStorage = create<CartState>()(
       },
       chainToPayoutToken: ethOnlyPayoutTokens,
       setPayoutTokenForChain: (chainId: ChainId, payoutToken: PayoutToken) => {
+        if (!Object.values(ChainId).includes(chainId)) {
```

Comment:
Is setting an invalid payout token for a chain a valid operation? If not, we might want some degree of warning on the `console.warn` to `throw` spectrum.

Reply:
It's an invalid op. There is no use-case where this should happen. Added a console.warn.